### DISCUSSION TITLE
F-Droid Ungoogled Chromium Android

### DISCUSSION THREAD

**Discussion by ghost** [ungoogled-software/ungoogled-chromium-android#122] [General]
Ungoogled Chromium in Android F-Droid in Still V 97.

Would you please update it.
@wchen342

Github 99 available 26 days ago.
![ung](https://user-images.githubusercontent.com/103345044/162618312-9a16bac1-3a2e-4e78-8d49-ab77d45fd8ca.png)
 
But F-Droid Repo still not now.
I only update in ungoogled chromium f-droid repo.

**Comment by wchen342** [comment]
I probably won't be able to update things anytime soon. I will put out a notice ASAP. At the meantime I will suggest using [one of these browsers](https://github.com/Tobi823/ffupdater#firefox-updater) instead.

**Reply by ghost** [reply]
Okay

**Comment by r-a-y** [comment]
You can find a 3rd-party build of Ungoogled Chromium here: https://github.com/macchrome/droidchrome/releases/latest
